docs/proposals/capacity-planning-refactoring.md

```diff
@@ -0,0 +1,100 @@
+# Proposal: Refactoring of capacity planning
+
+Last updated: 09.08.2021
+
+
+## Abstract
+
+Refactor logic of AC reservation to consider cases for 2 or more LVG PVs
+
+## Background
+
+Currently, 2 or more LVG PVs can be assigned to different disks on the reservation phase. It leads to the ineffective space usage.
+
+Causes:
+1. Reservation Controller doesn't consider on the planning phase, that LVG volumes should use one device. 
+   If ACR reserves AC for LVG, other ACR won't be able to use the same drive for LVG.
```

Comment:
> If ACR reserves AC for LVG

If ACR reserves non-LVG AC for LVG

> ACR won't be able to use the same drive for LVG

until reservation is release or AC is converted

Reply:
updated